.bootstrap/.core/skills/cypilot/scripts/cypilot/commands/update.py

```
@@ -350,3 +374,67 @@ def _read_project_name(config_dir: Path) -> Optional[str]:
+
+    Returns True if user acknowledged (or non-interactive), False if declined.
+    """
+    missing = sorted(
+        (v, ref_whatsnew[v]) for v in ref_whatsnew
+        if v not in core_whatsnew
+    )
+    if not missing:
+        return True
+
+    sys.stderr.write(f"\n{'=' * 60}\n")
+    sys.stderr.write(f"  What's new in Cypilot\n")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)